settings.html

```diff
@@ -70,6 +70,10 @@ <h3>{{ theme }}</h3>
       <label for='hide-menu-bar'>{{ hideMenuBar }}</label>
     </div>
     {{ /isHideMenuBarSupported }}
+     <div class='tray-setting'>
```

Comment:
should this class be renamed if the reference to the tray was removed from the messaging for the setting?